package.json

```diff
@@ -26,7 +26,8 @@
     "grunt": "^1.0.1",
     "grunt-build-control": "^0.7.1",
     "grunt-cli": "^1.2.0",
-    "kalastatic": "^3.1.0"
+    "kalastatic": "^3.1.0",
+    "sass-lint": "^1.12.0"
```

Comment:
Josh bringing the SASS

![tumblr_m8q67ag7lx1qd1d32o1_500](https://user-images.githubusercontent.com/25086/31198447-5b7bc1da-a922-11e7-98a0-5476f6ade357.gif)